Resources/doc/form.rst

```diff
@@ -103,7 +105,7 @@ Next, create the form for the ``User`` model::
             ));
         }
 
-        public function setDefaultOptions(OptionsResolverInterface $resolver)
+        public function configureOptions(OptionsResolver $resolver)
         {
             $resolver->setDefaults(array(
                 'data_class' => 'Acme\AccountBundle\Document\User',
```

Comment:
While you're at it, could you change this to use the `::class` constant as well?